security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -13,6 +13,26 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-30"]
+== OpenShift File Integrity Operator 0.1.30
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.30:
+
+* link:https://access.redhat.com/errata/RHBA-2022:5538[RHBA-2022:5538 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+
+[id="file-integrity-operator-0-1-30-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The File Integrity Operator is now supported on the following architectures:
++
+** IBM Power
+** IBM Z and LinuxONE
```

Comment:
Just wondering if Z enablement necessarily means LinuxONE (perhaps it does).

Reply:
Yes, I confirmed with our IBM partner engineer. They specifically requested this formatting. Thanks for confirming :+1: